charter.md

```diff
@@ -0,0 +1,55 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+## Governance
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition. Decisions can be made both in the repository and in meetings, as long as consensus can be established. By default, the group will operate in public. However, specific decision-making processes can be made member-confidential by defining them as such in a process document published with the consensus of the team.
```

Comment:
```suggestion
The team makes decisions based on consensus. A decision has consensus if it has support from at least two participating organizations and no opposition.

The team operates in public, except when otherwise allowed by this charter.

The team may publish a process document to define how it will develop a given metric. These process documents are adopted by a decision of the team. These process documents may define parts of their operation to be confidential to the team.
```